WordPress/Classes/Models/Comment/Comment+CoreDataClass.swift

```diff
@@ -81,24 +77,21 @@ public class Comment: NSManagedObject {
     }
 
     func canReply() -> Bool {
-        if let readerPost = post as? ReaderPost {
-            return readerPost.commentsOpen && ReaderHelpers.isLoggedIn()
```

Comment:
`ReaderHelpers.isLoggedIn` – this was super slow as it contains Core Data access with `performAndWait`. This check is not necessary as the app doesn't allow you to view Reader without being logged in.